agent-inject/agent/annotations_test.go

```diff
@@ -538,6 +538,55 @@ func TestSecretTemplateFileAnnotations(t *testing.T) {
 }
 
 func TestSecretCommandAnnotations(t *testing.T) {
```

Comment:
It looks like this test and the next have their names flipped.
```suggestion
func TestSecretPermissionAnnotations(t *testing.T) {
```